activerecord/lib/active_record/timestamp.rb

```diff
@@ -28,6 +28,10 @@ module ActiveRecord
   #
   #   ActiveRecord::Base.time_zone_aware_types = [:datetime]
   #
+  # NOTE: If you are using PostgreSQL you are likely to add <tt>tsrange</tt> and <tt>tstzrange</tt> to timezone aware types:
+  #
+  #   ActiveRecord::Base.time_zone_aware_types += [:tsrange, :tstzrange]
+  #
```

Comment:
Ok, we can re-write this note as an example, but unless PostgreSQL adapter is a part of ActiveRecord, I think, we should provide specific information, not generic. This can help us to avoid bug reports/ issues like "Help! tsrange not working" etc.